lib/lz4.c

```diff
@@ -517,7 +517,7 @@ static unsigned LZ4_NbCommonBytes (reg_t val)
     assert(val != 0);
     if (LZ4_isLittleEndian()) {
         if (sizeof(val) == 8) {
-#       if defined(_MSC_VER) && (_MSC_VER >= 1800) && defined(_M_AMD64) && !defined(LZ4_FORCE_SW_BITCOUNT)
+#       if defined(_MSC_VER) && (_MSC_VER >= 1800) && (defined(_M_AMD64) && !defined(_M_ARM64EC)) && !defined(LZ4_FORCE_SW_BITCOUNT)
```

Comment:
Could you please add a one-line comment explaining why `_M_ARM64EC` must be exclude ?
Essentially a summary of what you already explain in the PR, just hosted directly in the code source.

Reply:
Thanks, I just added comments.